htcanalyze/htcanalyze.py

```diff
@@ -337,19 +291,20 @@ def log_to_dict(self, file: str, sec: int = 0
                 memory_requested = event.get('RequestMemory', np.nan)
                 memory_allocated = event.get('Memory', np.nan)
 
-                # put the data in the dict
-                res_dict = {
-                    "Resources": ["Cpu", "Disk", "Memory"],
-                    "Usage": np.array(
-                        [cpu_usage, disk_usage, memory_usage],
-                        dtype=np.float64),
-                    "Requested": np.array(
-                        [cpu_requested, disk_requested, memory_requested],
-                        dtype=np.float64),
-                    "Allocated": np.array(
-                        [cpu_allocated, disk_allocated, memory_allocated],
-                        dtype=np.float64)
-                }
+                # create list with resoucres
+                resource_list = [Resource("Cpu",
```

Comment:
Is there a reason you're calling it `"Cpu"` instead of `"cpu"`? I prefer doing everything lower case to prevent comparison issues with upper- and lower-case values.

Reply:
Just because then I don't have to make it upper case when printing it. I prefer lower case too, but when I print it, it should be in upper case.

Reply:
It wouldn't harm though. What do you think, Should I just change it, when it's printed ?

Reply:
I will change the `name` attribute to `description`. In that way it makes more sense to have upper/lower case and spaces in the string.

Reply:
There is no comparison yet and if there will be, it would be easier to then compare it that way, instead of changing the description string, as this is getting printed and also concatenated ( see `create_avg_on_resources`).
So in this case I will leave it as it is.

Reply:
To modify string capitalization, you could use `str.lower()`, `str.upper()` and `str.capitalize()` for visualization purposes. That way it doesn't matter how you write it initially. But since CPU is an abbreviation of 'central processing unit', it should all be in caps anyway so you can do `Resource.name.upper()`.

Reply:
In this case I will leave it as it is ( but change Cpu -> CPU), but I will consider this in other cases.
Thanks